src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/PreferStreamAsyncMemoryOverloadsTestBase.cs

```diff
@@ -315,12 +315,7 @@ public static IEnumerable<object[]> VisualBasicUnnamedArgumentsPartialBufferTest
         }
 
         public static IEnumerable<object[]> VisualBasicUnnamedArgumentsFullBufferTestData()
-        {
-            yield return new object[] { "buffer, 0, buffer.Length",
-                                        "buffer" };
-            yield return new object[] { "buffer, 0, buffer.Length, CancellationToken.None",
-                                        "buffer, CancellationToken.None" };
-        }
+            => CSharpUnnamedArgumentsFullBufferTestData();
```

Comment:
Feels better to just delete `VisualBasicUnnamedArgumentsFullBufferTestData` rename `CSharpUnnamedArgumentsFullBufferTestData` to `UnnamedArgumentsFullBufferTestData`